rpc/handler.go

```diff
@@ -477,36 +511,7 @@ func (h *handler) runMethod(ctx context.Context, msg *jsonrpcMessage, callb *cal
 		stream.WriteMore()
 	}
 	stream.WriteObjectField("result")
-	_, err := callb.call(ctx, msg.Method, args, stream)
-	if err != nil {
-		//return msg.errorResponse(err)
-
-		stream.WriteMore()
-		stream.WriteObjectField("error")
-		stream.WriteObjectStart()
-		stream.WriteObjectField("code")
-		ec, ok := err.(Error)
-		if ok {
-			stream.WriteInt(ec.ErrorCode())
-		} else {
-			stream.WriteInt(defaultErrorCode)
-		}
-		stream.WriteMore()
-		stream.WriteObjectField("message")
-		stream.WriteString(fmt.Sprintf("%v", err))
-		de, ok := err.(DataError)
-		if ok {
-			stream.WriteMore()
-			stream.WriteObjectField("data")
-			data, derr := json.Marshal(de.ErrorData())
-			if derr == nil {
-				stream.Write(data)
-			} else {
-				stream.WriteString(fmt.Sprintf("%v", derr))
-			}
-		}
-		stream.WriteObjectEnd()
-	}
+	callb.call(ctx, msg.Method, args, stream)
```

Comment:
please don't loose error returned by this method. also pass it to HandleError

Reply:
Shoot! I was on the fence about that one but now realize that's important for the other higher level errors that may occur (before looping through transactions). I'll get this added to the PR asap!

Reply:
That change (and an omission from the rpcdaemon22 port) has been made!